### ISSUE TITLE
Getting prompted to upgrade above 45 when using 48

### ISSUE THREAD

**Issue by chuckharmston** [mozilla/testpilot#1418]
Seeing this on stage as of our push this evening:

<img width="1085" alt="screen shot 2016-09-22 at 9 40 17 pm" src="https://cloud.githubusercontent.com/assets/23885/18773537/47ddadc0-810d-11e6-8c87-382910f61af2.png">

cc @johngruen, I think you touched this.

``` json
{
  "source": "https://github.com/mozilla/testpilot",
  "commit": "1bf0f9b76895565efd646e0ed13dab2a09573fd8",
  "build": "https://circleci.com/gh/mozilla/testpilot/1071",
  "version": "2016-09-22"
}
```

**Comment by johngruen** [comment]
@chuckharmston check dev, i think @dannycoates fixed this bad boy

**Comment by johngruen** [comment]
Fixed in #1415
